misc/navigation.py

```diff
@@ -3,19 +3,18 @@
 ctx = Context("navigation")
 
 keymap = {
+    # Requires activation of System Preferences -> Shortcuts -> Input Sources
+    # -> "Select the previous input source"
+    "change language": Key("ctrl-space"),
+    # moving
+    "tarsh": Key("shift-tab"),
     # scrolling
     "scroll down": [Key("down")] * 30,
     "scroll up": [Key("up")] * 30,
     # NB home and end do not work in all apps
     "(scroll way down | doomway)": Key("cmd-down"),
     "(scroll way up | jeepway)": Key("cmd-up"),
-    "page up": [Key("pageup")],
-    "page down": [Key("pagedown")],
-    # searching
```

Comment:
these should stay, std.py should be removed.

Reply:
So page up and down, should stay in navigation.py? In addition, to the (pageup | page up) fix in basic keys?